pymongo/message.py

```diff
@@ -1119,6 +1119,10 @@ def _check_doc_size_limits(
         new_message_size = total_ops_length + total_ns_length + op_length + ns_length
         # We have enough data, return this batch.
         if new_message_size > max_doc_sequences_bytes:
+            if idx == 0:
+                raise InvalidOperation(
+                    "No operations can be executed -- given operation is too large"
+                )
```

Comment:
We should consider raising the same error as collection.bulk_write in this case.